Monika After Story/game/script-topics.rpy

```diff
@@ -4875,13 +4875,13 @@ label monika_panties:
     m 3rksdlb "Someone who maybe stole your pen?"
     m 1eua "But, to each their own I guess, I won't judge too much."
     if isFuture (evhand.event_database['anni_1year']):
```

Comment:
remove space between isFuture and the ().  also should be 'anni_1' not' anni_1year'